legend-depot-artifacts-refresh/src/main/java/org/finos/legend/depot/store/artifacts/services/file/FileGenerationHandlerImpl.java

```diff
@@ -134,6 +136,11 @@ public MetadataEventResponse refreshProjectVersionArtifacts(StoreProjectData pro
         return response;
     }
 
+    private String getExtensionKeyFromGeneration(String path, String entityPath)
+    {
+        return Arrays.stream(path.replace(PATH + entityPath + PATH, "").split(PATH)).collect(Collectors.toList()).get(0);
```

Comment:
can tyou rename the constant PATH to PATH_SEPARATOR..  i know is not your chage but code would read better